### ISSUE TITLE
Question - Did anyone able to book 45+ using this script ?

### ISSUE THREAD

**Issue by xxxAng** [bombardier-gif/covid-vaccine-booking#393]
Did anyone able to book 45+ using this script ?
I was not successful, it was only showing 18+ slots and when tried to book gave error like "Minimum age criterion dint match".

**Comment by Nakul93** [comment]
That’s rather strange. Technically it should not proceed with the booking since it didn’t match the min age criteria. Do you have the latest code?

**Comment by xxxAng** [comment]
Yup @Nakul93  can you confirm you were able to book 45+ candidate with latest script ?

Or anyone ?

**Comment by Nakul93** [comment]
Yeah I was able to book 3-4 days back

**Comment by xxxAng** [comment]
ok thanks
